src/Command/Argument/TimeSeries/CommonArguments.php

```diff
@@ -44,6 +45,24 @@ public function retentionMsecs(int $retentionPeriod): self
         return $this;
     }
 
+    /**
+     * Ignore samples with given time or value difference.
+     *
+     * @param  int   $maxTimeDiff Non-negative integer value in milliseconds
+     * @param  float $maxValDiff  Non-negative float value
+     * @return $this
+     */
+    public function ignore(int $maxTimeDiff, float $maxValDiff): self
+    {
+        if ($maxTimeDiff < 0 || $maxValDiff < 0) {
+            throw new UnexpectedValueException('Ignore does not accept non-positive values');
```

Comment:
0 can also be considered non-positive? Maybe use the word "negative" instead of "non-positive".